revscoring/scorers/svc.py

```diff
@@ -131,6 +125,21 @@ def test(self, values_labels, comparison_class="auto"):
             'auc': auc(fpr, tpr)
         }
     
+    def _balance_labels(self, values_labels):
+        groups = defaultdict(lambda: [])
+        for feature_values, label in values_labels:
+            groups[label].append(feature_values)
+        
+        max_label_n = max(len(groups[label]) for label in groups)
+        
+        new_values_labels = []
+        for label in groups:
+            new_values_labels.extend((random.choice(groups[label]), label)
+                                      for i in range(max_label_n))
```

Comment:
What does this loop do? Does it duplicate a few random examples of the class(es) with the a small number of representatives until we have all classes with the same size? Or something else?
Maybe add a short comment explaining the idea?